resources/dicts/relationship_events/normal_interactions/neutral.json

```diff
@@ -47,8 +46,7 @@
 			"m_c asks r_c to remind {PRONOUN/m_c/object} of who's already gone on patrol today.",
 			"m_c asks r_c about a certain rule of the Warrior Code that {PRONOUN/m_c/subject} forgot.",
 			"m_c watches r_c training for something and nods approvingly at {PRONOUN/r_c/poss} progress.",
-			"m_c should probably break up the scuffle between r_c and a Clanmate, but it's entertaining.",

```

Comment:
Removed

Reply:
Removed this to save it for a group interaction if I do those in the future :)!!